spaceship/lib/spaceship/client.rb

```diff
@@ -881,6 +879,12 @@ def send_request(method, url_or_path, params, headers, &block)
       end
     end
 
+    def handle_401(response)
```

Comment:
Just a nit-pick, `response` is unused in the method body. Either remove the parameter or use it in the `warn` ?

Reply:
Need to keep the response there for the methods that override this even though its not being used here 😔 Ruby will yell at us otherwise 🙄